source/index-management/tests/createSearchIndex.yml

```diff
@@ -0,0 +1,62 @@
+description: "createSearchIndex convenience helper"
+schemaVersion: "1.10"
+createEntities:
+  - client:
+      id: &client0 client0
+      useMultipleMongoses: false
+      observeEvents:
+        - commandStartedEvent
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: *database0
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: *collection0
+
+runOnRequirements:
+  - minServerVersion: "7.0.0"
+    topologies: [ replicaset, load-balanced, sharded ]
+    serverless: forbid
+
+tests:
+  - description: "no name provided for an index definition"
+    operations:
+      - name: createSearchIndex
+        object: *collection0
+        arguments:
+          definition: { definition: &definition { mappings: { dynamic: true } }  }
```

Comment:
```suggestion
          definition: { definition: &definition { mappings: { dynamic: true } } }
```

Reply:
We do not. I'd totally be down for integrating a YAML linter into our CI pipeline, though.